source/tester/tester-tests.adb

```diff
@@ -33,7 +33,7 @@ with Spawn.Processes.Monitor_Loop;
 
 package body Tester.Tests is
 
-   Max_Wait : constant := 4_000;
+   Max_Wait : constant := 40_000;
```

Comment:
I think this should be changed back to 4_000, in light of the new commit for the testsuite

Reply:
Right